ansible_collections/arista/cvp/plugins/modules/cv_container.py

```diff
@@ -261,6 +304,10 @@ def tree_build_from_list(containers):
     ----------
     containers : dict, optional
         Container topology to create on CVP, by default None
+    root: string, optional
+        Name of container to consider as root for topology, by default Tenant
+    debug : bool, optional
+        Activate debug logging, by default False
```

Comment:
i think there's an easier way to debugging. something like https://github.com/ansible/ansible/blob/5b3526535c9e2f9e117e73d8fbea24b6f9c849ef/lib/ansible/modules/monitoring/logicmonitor.py#L519
which works like this :
https://github.com/ansible/ansible/blob/devel/lib/ansible/module_utils/basic.py#L1861

it looks cleaner at least than passing this variable around every function

Reply:
Thanks for these pointers. Looks like definitely a better approach and will make it part of the next version for all collection's modules.

Reply:
Track with issue #124